http/src/main/resources/reference.conf

```diff
@@ -85,7 +83,7 @@ dataproc {
 }
 
 gce {
-  customGceImage = "projects/broad-dsp-gcr-public/global/images/custom-leo-gce-cos-image-da32384"
+  customGceImage = "projects/broad-dsp-gcr-public/global/images/leo-gce-image-qi-2021-07-15-14-25-08"
```

Comment:
custom images are generated locally becuz I haven't been able to run the jenkins job successfully